### PR TITLE
[GH-1264] Fix qri stats pretty printing

### PR DESCRIPTION
**Pull request by VarsMolta**
This PR adds effective pretty printing to the `qri stats` command.

### CODE REVIEW

File: cmd/stats.go [hunk 1]
```diff
@@ -73,7 +76,19 @@ func (o *StatsOptions) Run() (err error) {
...
 
-	r.StatsBytes = append(r.StatsBytes, byte('\n'))
-	printInfo(o.Out, string(r.StatsBytes))
+	var buffer []byte
+	if !o.Pretty {
+		buffer = append(r.StatsBytes, byte('\n'))
+	} else {
+		// stats is already in JSON format, so just needs pretty printing
+		buf := new(bytes.Buffer)
+		err = json.Indent(buf, []byte(r.StatsBytes), "", "  ")
```
[the remark this answers is not in the document]
Comment (VarsMolta) [reply cmd/stats.go:85]: 👍

**Comment by VarsMolta** [comment]
> Looks great! The only thing I'd ask is that you amend your commit message to match our [commit message guidelines](https://github.com/qri-io/qri/blob/master/DEVELOPERS.md#-git-commit-guidelines)
> 
> I'd re-word this commit to look like this:
> 
> ```
> feat(cmd): add pretty flag to stats command
> 
> use json.Indent to add indentation to stats data returned from lib
> 
> closes #1264
> ```
> 
> but feel free to match the commit message to your own tone. Once that change is up, let's get this merged!

Done!

File: cmd/stats.go [hunk 2]
```diff
@@ -1,6 +1,9 @@
 package cmd
 
 import (
+	"bytes"
+	"encoding/json"
+	"fmt"
 	"github.com/qri-io/ioes"
```
[the remark this answers is not in the document]
Comment (VarsMolta) [reply cmd/stats.go:8]: Done

**Comment by b5** [comment]
I ended up having to squash-merge because `style:` isn't one of our topic names, but no worries.

Great stuff @VarsMolta, thanks so much for your contribution!

**Pull request merged**